west.yml

```diff
@@ -44,7 +44,7 @@ manifest:
       revision: 99129c5efc907ea613c4b73ccff07581feb58a7a
       path: modules/lib/civetweb
     - name: esp-idf
-      revision: 6835bfc741bf15e98fb7971293913f770df6081f
+      revision: 517c4cf5dc322848f3196e48828936eb2eb4eb56
```

Comment:
You probably want this to be `pull/3/head`

https://docs.zephyrproject.org/latest/guides/modules.html?highlight=modules#process

Reply:
Fixed